integration-tests/test/rnode.py

```diff
@@ -364,24 +413,31 @@ def make_bootstrap_node(
     command_timeout: int,
     allowed_peers: Optional[List[str]] = None,
     mem_limit: Optional[str] = None,
+    cli_flags: Optional[AbstractSet] = None,
     cli_options: Optional[Dict] = None,
     mount_dir: Optional[str] = None,
+    wallets_file: Optional[str] = None,
 ) -> Node:
     key_file = get_absolute_path_for_mounting("bootstrap_certificate/node.key.pem", mount_dir=mount_dir)
     cert_file = get_absolute_path_for_mounting("bootstrap_certificate/node.certificate.pem", mount_dir=mount_dir)
 
     container_name = make_bootstrap_name(network)
 
+    container_command_flags = set([
```

Comment:
I feel it'd be good to use ordered set for command line flags, but AFAIK no options in rnode currently depend on order, so this is fine.